### DISCUSSION TITLE
Wth happened,

### DISCUSSION THREAD

**Discussion by toxicsloth** [SEVENNetworks/AdClear-Content-Blocker-Open-Source-Issue-Tracker#2] [General]
Used to like this blocker, as it gave phone wide protection from ads, but switch phones and now it is just a browser plug in, and blocks nothing otherwise? Yet another example of an app that used to be good, but now does nothing it is expected to.

**Comment by DustinMorgan** [comment]
Hey @toxicsloth, we'd like to look a bit more into this. What device model and version of Android are you on? Can you give us some apps are you seeing the ads in so we can try and reproduce them?
